ui/core-react/src/core-react/tabs/Tabs.tsx

```diff
@@ -43,7 +43,7 @@ export interface TabsProps
     CommonProps {
   /** Text shown for each tab
    * @public */
-  labels: Array<string | TabLabel>;
+  labels: Array<string | TabLabel | any>;
```

Comment:
Remove public tag. Adding `any` to a public type is a breaking change, which we can't do.
```
/** Text shown for each tab */
labels: Array<string | TabLabel>;
```